docs/properties/recommended_optional/property_distribution.rst

```diff
@@ -31,30 +29,47 @@ Collections of files should be either using an archive format or a BagIt folder
      </distribution>
    </distributions>
 
-.. _21.a:
 
-21.a mediaType
+.. _21.1:
+
+21.1 file
 ~~~~~~~~~~~~~~~~~~~~
 
+**Occurrences:** 1-n
+
+**Definition:** A container in a computer system for storing information.
+
+**Allowed values, examples, other constraints:**
+
+If Distribution is used, at least one file is mandatory.
+
+A file can either represent the entirety of the distribution, or a part of it. Collections of files can be described either using a single file (e.g., an archive format or a BagIt folder structure) or as multiple files. See :doc:`/guidance/distribution` for recommendations.
```

Comment:
In my opinion, this first release of **Distribution** should focus on the simplest case (fully informationally equivalent distributions and the use of archive files). **The WG sub-group worked the whole time with that in mind**. Future releases (4.6 or 5.0) could address more complex use cases (as the WG did with **Data Service**). As we have seem in the proposals, they have been floated so far, those more complex use cases seem to be around notes and recommendations. So it might be likely that early adopters won’t see themselves affected.

I would recommend that the WG look at establishing discussions with the DCAT find a robust solution for those more complex cases.

Reply:
Agreed, we want to keep distributions fully equivalent. The proposal to have files within a distribution represented is meant to encourage that usage. Otherwise, it will be more likely that users will repeat the distribution property to represent non-equivalent distributions.

Reply:
I am with Kelly. The change of DataCite metadata is tracked, so there is no need to imply it in distribution. The aspect with the comparison of the checksums also seems plausible to me. So, in order not to inflate the scheme unnecessarily, it probably makes sense to omit lastUpdated for the current release.

Edit: Doesn't also Bagit cover the issue of checksums for files (manifest)?